test/core/libp2p/peer/peer_info_test.cpp

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string_view>
+
+#include <gtest/gtest.h>
+#include "libp2p/peer/peer_info.hpp"
```

Comment:
not consistent, in some places the tested module included first as in [peer_id_test](https://github.com/soramitsu/kagome/pull/56/files#diff-55c19d071d02be1f101edb968513915cR6) in some places like here it is the last included module. Please fix

Reply:
Well, this is because of clang-format, it does strange things, sometimes sorting the test headers by alphabet, sometimes not. I assume gtest include should be the first